examples/infinity_feedliker_example.py

```diff
@@ -0,0 +1,21 @@
+"""
+    instabot example
+
+    Workflow:
+        Like rescent medias from your timeline feed.
+"""
+
+import sys
+import os
+import time 
+
+sys.path.append(os.path.join(sys.path[0], '../'))
+
+from instabot import Bot
+
+bot = Bot()
+bot.login()
+
+while True:
+	bot.like_timeline()
+	time.sleep(1*60)
```

Comment:
@stasfilin ok, what I must write if I want 2 mins 120 or 2*60, of 3 min 180(3*60) what most user friendly?